### DISCUSSION TITLE
[General] <Gas fee when upgrade contract on zkSync Era Mainnet>

### DISCUSSION THREAD

**Discussion by ngocthachcs** [zkSync-Community-Hub/zksync-developers#206] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

I'm upgrading the DID NFT contract.
When redeploying, the gas fee is very high ~$35, previously the gas fee was only $5 - $15.

### What is the specific issue or error you're encountering?

High gas fee

### Can you share the error messages or logs you're receiving, if any?

body: '{"jsonrpc":"2.0","error":{"code":3,"message":"insufficient funds for gas + value. balance: 15204926812500000, fee: 16819506500000000, value: 0","data":"0x"},"id":60}\n',

### Have you made any recent changes to the contract before encountering this issue?

I just updated the data which is the new price of the NFT.

### Are there any external libraries or contracts that your contract interacts with?

N/A

### Can you provide the relevant portions of your contract code where the issue is occurring?

N/A

### Have you tried to isolate the problem, and if so, what were the results?

N/A

### What steps have you already taken to try to resolve the issue?

N/A

**Comment by dutterbutter** [comment]
Can you provide a link to the contract / repo you are deploying?
